scripts/fiber/record-tests

```diff
@@ -106,12 +106,14 @@ function runJest(maxWorkers) {
 function formatResults(runResults, predicate) {
   const formatted = [];
   runResults.testResults.forEach((fileResult) => {
-    const file = path.relative(root, fileResult.testFilePath);
+    const filePath = path.relative(root, fileResult.testFilePath);
+    // on windows, we still want to output forward slashes
+    const unixFilePath = filePath.replace(/\\/g, '/');
```

Comment:
No, that still uses windows line endings
```
path.normalize('C:\\temp\\\\foo\\bar\\..\\');
// Returns: 'C:\\temp\\foo\\'
```
https://nodejs.org/api/path.html#path_path_normalize_path